### PR TITLE
TSK-944: Improve performance of AsyncUpdateJobIntTest

### PR DESCRIPTION
**Pull request by benjamineckstein**

### CODE REVIEW

File: rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java [hunk 1]
```diff
@@ -58,86 +51,75 @@
...
-        con.setDoOutput(true);
-        con.setRequestProperty("Content-Type", "application/json");
-        BufferedWriter out = new BufferedWriter(new OutputStreamWriter(con.getOutputStream()));
-        out.write(updatedClassification);
-        out.flush();
-        out.close();
-        assertEquals(200, con.getResponseCode());
-        con.disconnect();
+        template.put(
+            server + "/api/v1/classifications/{classificationId}",
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java]: extract /api/v1/classifications to constant?

File: rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java [hunk 2]
```diff
@@ -58,86 +51,75 @@
...
-    private static final Logger LOGGER = LoggerFactory.getLogger(AsyncUpdateJobIntTest.class);
-    String server = "http://127.0.0.1:";
-    RestTemplate template;
-    HttpEntity<String> request;
-    HttpHeaders headers = new HttpHeaders();
-    @LocalServerPort
-    int port;
+    @Autowired JobScheduler jobScheduler;
+
+    @Autowired Environment env;
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java]: please make a newline after each annotation. thanks!

File: rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java (hunk 1 above)
Comment (benjamineckstein) [reply rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java]: next step ...  want to improve urls on another ticket

File: rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java (hunk 2 above)
Comment (benjamineckstein) [reply rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java]: done

**Pull request merged**